### DISCUSSION TITLE
Cookies should be sent on boot up.

### DISCUSSION THREAD

**Discussion by engineerhead** [maxcountryman/tower-sessions#46] [Ideas]
Hey! From what I have understood, cookies(session id) are sent only if something is changed(modified) in session store.  Shouldn't they be sent on middleware boot up as is the case in express.js' session handling.

**Comment by maxcountryman** [comment]
It's intentional that they aren't sent unless the session has a non-empty value.

This mirrors other session implementations, like Django's for example.
